pymess/backend/dialer/daktela.py

```diff
@@ -40,6 +40,10 @@ def _update_dialer_states(self, messages):
             })
             resp_message_state = resp_json['result']['statuses'][0]['name'] if len(
                 resp_json['result']['statuses']) else resp_json['result']['action']
+            whole_message_heard = resp_json['result']['customFields']['whole_message_heard'][0] == 'Yes'
+            if 'statuses_5c1394fa3ea76819966092' == resp_message_state:
+                if whole_message_heard:
```

Comment:
Proč to nespojíš andem?

Reply:
povodne som to tak mal, ale kvoli niecomu som tom menil. uz si nespominam na dovod... vratil som to nazad